arch/arm/include/setjmp.h

```diff
@@ -25,6 +25,7 @@
  * Included Files
  ****************************************************************************/
 
+#include <nuttx/compiler.h>
```

Comment:
should we mvoe it after nuttx/config.h?

Reply:
Does it make a difference?  
Are there any rules in NuttX for the includes order?

Reply:
Since config.h include the important setting, all most source always include config.h first.